tezedge_state/src/state/connected_peers.rs

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) SimpleStaking, Viable Systems and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+
+use crypto::crypto_box::PublicKey;
+use getset::{CopyGetters, Getters};
+use slog::Logger;
+use std::collections::{BTreeMap, VecDeque};
+use std::io::{Read, Write};
+use std::time::{Duration, SystemTime};
+
+use crate::chunking::{
+    EncryptedMessageWriter, MessageReadBuffer, ReadMessageError, WriteMessageError,
+};
+use crate::peer_address::PeerListenerAddress;
+use crate::state::pending_peers::HandshakeResult;
+use crate::state::ThrottleQuota;
+use crate::{PeerAddress, PeerCrypto, Port};
+use tezos_messages::p2p::encoding::peer::{PeerMessage, PeerMessageResponse};
+use tezos_messages::p2p::encoding::prelude::NetworkVersion;
+
+/// Peer who have undergone handshake.
+#[derive(Getters, CopyGetters, Debug, Clone)]
+pub struct ConnectedPeer {
+    // #[get = "pub"]
```

Comment:
@binier 
hmm, I would remove those commented annotations,
and internal state shouldnt be exposed as `pub`, if we dont want getters here, better use `pub(crate)`

Reply:
That is an internal structure used only in `TezedgeState`. It's goal is to guard against such access. Anyways removed to comments.